libnavigation-core/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
     implementation project(':libdirections-session')
     implementation project(':libtrip-session')
     implementation(project(':libnavigator'))
+    implementation(project(':libdirections-offboard'))
```

Comment:
Should `core` depend on `offboard` ?

Reply:
This "core" module is what will become `libandroid-navigation` `1.0`, so will likely include all of the modules. We created a separate module for early integration and testing.